climada/util/test/test_config.py

```diff
@@ -61,6 +61,12 @@ def test_substitute(self):
         self.assertEqual(conf.a._root, conf._root)
         self.assertEqual(conf.a.str(), 'https://host/page.domain')
 
+    def test_missing(self):
+        with self.assertRaises(AttributeError) as ve:
+            CONFIG.hazard.fire_fly.population.str()
+        self.assertIn("there is no 'fire_fly' configured for 'hazard'", str(ve.exception))
```

Comment:
What does one get for a message for `CONFIG.random_module` and for `CONFIG.hazard.tropical_cyclone.some_file`? Should this also be tested maybe?

Reply:
This:
- "there is no 'random_module' configured for 'climada.CONFIG' ..."
- "there is no 'some_file' configured for 'tropical_cyclone' ..."
- 
The latter should not be tested as there is nothing such a test could contribute. The former possibly to some extent as the 'climada.CONFIG' is something special. I wouldn't but if you if you think it's worth it I can add it.

Reply:
Well, the given that there is nothing currently, but it would be possible (and in particular a user can add it right?) I would suggest adding it since it is very little extra work.

Reply:
👍 sure.
Just for understanding - what do you mean by 
> Well, the given that there is nothing currently, but it would be possible (and in particular a user can add it right?)

?